### PR TITLE
Pollish up chinese translation (intro2jl-short)

### PR DESCRIPTION
**Pull request by inkydragon**
`intro-to-julia/short-version`

- [x] 00.上手Jupyter_notebook
- [x] 01.了解Julia
- [x] 02.Julia中的线性代数
- [x] 03.包的使用
- [x] 04.Julia绘图简介
- [x] 05.Julia很快
- [x] 06.多重派发

all code test in julia 1.0.5 & sync with english version.

### CODE REVIEW

**Comment by NHDaly** [comment]
Awesome! Thanks so much for this contribution @inkydragon!!

Given that my Mandarin is just not up to the level needed to meaningfully review this, I'm inclined to just approve it.

Does anyone else want to review first? @Roger-luo?
Thanks!

File: .nbexports/zh-cn/intro-to-julia-ZH/简短版/02.Julia中的线性代数.jl [hunk 1]
```diff
@@ -1,65 +1,77 @@
...
 # ------------------------------------------------------------------------------------------
 
 x = fill(1.0, (3))
 
 # ------------------------------------------------------------------------------------------
-# 注意$A$的类型为Array{Int64,2}，而$x$的类型为Array{Float64,1}。Julia定义向量Vector{Type}的别名为Array{Type,1}，矩阵
-# Matrix{Type}的别名为Array{Type,2}。
+# 注意 $A$ 的类型为 `Array{Int64,2}`，而 $x$ 的类型为 `Array{Float64,1}`。
+#
+# Julia 定义向量 `Vector{Type}` 的别名为 `Array{Type,1}`，矩阵`Matrix{Type}` 的别名为 `Array{Type,2}`。
```
Comment (findmyway) [inline .nbexports/zh-cn/intro-to-julia-ZH/简短版/02.Julia中的线性代数.jl:32]: 我咋感觉这里反过来了？我一直以为 `Vector{Type}` 是 `Array{Type,1}`的alias，而`Matrix{Type}`是`Array{Type,2}`的alias，虽然语义上差别不大。

```suggestion
# Julia 定义 `Array{Type,1}` 的别名为向量 `Vector{Type}` ， `Array{Type,2}` 的别名为矩阵`Matrix{Type}` 。
```

File: .nbexports/zh-cn/intro-to-julia-ZH/简短版/06.多重派发.jl [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
-#
-# 多重派发让软件更*通用（generic）*，更*快*！
-#
-# #### 从熟悉的开始讲
+# ## 从熟悉的开始讲
 #
 # 从我们已经接触过的知识来理解多重派发。
 #
-# 我们可以声明一个Julia函数而不用给出传入变量的任何信息：
+# 我们可以声明一个 Julia 函数而不用给出传入变量的任何信息：
```
Comment (findmyway) [inline .nbexports/zh-cn/intro-to-julia-ZH/简短版/06.多重派发.jl:19]:
```suggestion
# 我们可以声明一个 Julia 函数而不用给出传入变量的任何类型信息：
```

File: .nbexports/zh-cn/intro-to-julia-ZH/简短版/06.多重派发.jl [hunk 3]
```diff
@@ -26,35 +30,35 @@ f(10)
 f([1, 2, 3])
 
 # ------------------------------------------------------------------------------------------
-# #### 指定传入变量的类型
+# ## 指定传入变量的类型
 #
 # 但是，我们*也可以*明确指定传入变量的类型。
```
Comment (findmyway) [inline .nbexports/zh-cn/intro-to-julia-ZH/简短版/06.多重派发.jl:35]:
```suggestion
# 但是，我们*也可以*显式地指定传入变量的类型。
```

File: .nbexports/zh-cn/intro-to-julia-ZH/简短版/06.多重派发.jl [hunk 4]
```diff
@@ -69,53 +73,52 @@ foo("hello", "hi!")
...
+# 想查看我们调用一个通用函数时派发的是哪个方法，我们可以用宏 `@which`：
 # ------------------------------------------------------------------------------------------
 
 @which foo(3, 4)
 
 @which 3.0 + 3.0
 
 # ------------------------------------------------------------------------------------------
-# 考虑到一个特别写给浮点数的方法派发给了`3.0 + 3.0`，生成的LLVM代码及其简练：
+# 考虑到一个特别写给浮点数的方法派发给了 `3.0 + 3.0`，生成的 LLVM 代码及其简练：
```
Comment (findmyway) [inline .nbexports/zh-cn/intro-to-julia-ZH/简短版/06.多重派发.jl:107]:
```suggestion
# 考虑到一个特别写给浮点数的方法派发给了 `3.0 + 3.0`，生成的 LLVM 代码极其简练：
```

File: .nbexports/zh-cn/intro-to-julia-ZH/简短版/02.Julia中的线性代数.jl (hunk 1 above)
Comment (inkydragon) [reply .nbexports/zh-cn/intro-to-julia-ZH/简短版/02.Julia中的线性代数.jl:32]: 确实有问题，看了一眼英文版是对的："Julia defines the aliases Vector{Type}=Array{Type,1} and Matrix{Type}=Array{Type,2}."
```
julia> Vector{Type}
Array{Type,1}

julia> Matrix{Type}
Array{Type,2}
```

**Comment by logankilpatrick** [comment]
@Roger-luo @findmyway does this look good to merge?

**Comment by Roger-luo** [comment]
LGTM

**Comment by logankilpatrick** [comment]
Thank you all! (Sorry for the merge delay)

**Pull request merged**